recipe/conda_build_config.yaml

```diff
@@ -12,3 +12,4 @@ libclang_soversion:
 # keep for use with root when it catches up
```

Comment:
For future reference, when adding root to a branch, you can delete this comment.